hack/release.sh

```
@@ -0,0 +1,563 @@
+success() { echo -e "${GREEN}✔${NC}  $*"; }
+warn()    { echo -e "${YELLOW}⚠${NC}  $*"; }
+error()   { echo -e "${RED}✖${NC}  $*" >&2; }
+header()  { echo -e "\n${BOLD}${CYAN}$*${NC}"; }
+dry_run() { echo -e "   ${YELLOW}[DRY RUN]${NC} $*"; }
+
+run_cmd() {
+    if $DRY_RUN; then
+        dry_run "$*"
+    else
+        info "Running: $*"
+        eval "$@"
```

Shellcheck (0.11.0)

[warning] eval negates the benefit of arrays. Drop eval to preserve whitespace/symbols (or eval as string).

(SC2294)